server/tournaments/index.ts

```diff
@@ -2172,6 +2220,73 @@ const commands: Chat.ChatCommands = {
 					return this.sendReply(`Usage: ${this.cmdToken}${this.fullCmd} <on|off>`);
 				}
 			},
+			recenttours(target, room, user) {
+				room = this.requireRoom();
+				this.checkCan('declare', null, room);
+				const num = parseInt(target);
+				if (!target ||
+					(!this.meansNo(target) && toID(target) !== 'forcedelete' &&
+						(isNaN(num) || num > 15 || num < 0))) {
+					return this.parse(`/help tour settings`);
+				}
+				if (!room.settings.tournaments) room.settings.tournaments = {};
+				if (!isNaN(num) && num <= 15 && num >= 1) {
+					if (room.settings.tournaments.recentToursLength === num) {
+						throw new Chat.ErrorMessage(`Recent tournament threshold is already set to ${num}.`);
+					}
+					room.settings.tournaments.recentToursLength = num;
+					room.saveSettings();
+					this.privateModAction(`Recent tournament threshold was set to ${num} by ${user.name}.`);
+					this.modlog('TOUR SETTINGS', null, `recent threshold: ${num}`);
+				} else if (this.meansNo(target) || toID(target) === 'forcedelete') {
+					if (room.settings.tournaments.recentToursLength) {
+						delete room.settings.tournaments.recentToursLength;
+						if (toID(target) === 'forcedelete') {
+							delete room.settings.tournaments.recentTours;
+							this.privateModAction(`Recent tournaments list was deleted by ${user.name}.`);
+							this.modlog('TOUR SETTINGS', null, `recent tours: delete`);
+						}
+						room.saveSettings();
+						this.privateModAction(`Recent tournament threshold was turned off by ${user.name}.`);
+						this.modlog('TOUR SETTINGS', null, `recent threshold: off`);
+					} else {
+						if (toID(target) !== 'forcedelete') {
+							throw new Chat.ErrorMessage(`Recent tournament threshold is already disabled.`);
+						}
+					}
+				} else {
+					this.sendReply(`Usage: ${this.cmdToken}${this.fullCmd} <number|off|forcedelete>`);
```

Comment:
Is there a reason that this isn't just in a normal help command? It has no aliases and I think most people can figure out the `/` vs. `!` distinction themselves.

Reply:
It is in the normal help command fwiw, it's just consistency with the other tours commands, I will likely refactor it later on

Reply:
If it's for consistency that's okay.